x-pack/test/security_solution_cypress/cypress/e2e/detection_response/rule_management/rule_actions/import_export/import_rules.cy.ts

```diff
@@ -20,7 +20,7 @@ import { RULES_MANAGEMENT_URL } from '../../../../../urls/rules_management';
 const RULES_TO_IMPORT_FILENAME = 'cypress/fixtures/7_16_rules.ndjson';
 const IMPORTED_EXCEPTION_ID = 'b8dfd17f-1e11-41b0-ae7e-9e7f8237de49';
 
-describe('Import rules', { tags: ['@ess', '@serverless'] }, () => {
+describe('Import rules', { tags: ['@ess', '@serverless', '@skipInServerlessMKI'] }, () => {
```

Comment:
What about here? Are we skipping it because it's flaky?

Import looks important enough to be in QA

Reply:
Cypress tests assert UI functionality. It's not critical since users can overcome it by using our API directly. Rule CRUD, import and rules table are critical features.